### PR TITLE
Add xLinux JenkinsFiles

### PR DESCRIPTION
**Pull request by AdamBrousseau**
- Add to nightly build
- Add PullRequest builds
- Add xLinux variables
- Update README

[skip ci]

Issue #1785

Signed-off-by: Adam Brousseau <adam.brousseau88@gmail.com>

### CODE REVIEW

**Comment by AdamBrousseau** [comment]
- [Build 8](https://ci.eclipse.org/openj9/job/Build-JDK8-linux_x86-64_cmprssptrs/4/)
- [Build 9](https://ci.eclipse.org/openj9/job/Build-JDK9-linux_x86-64_cmprssptrs/2/)
- [Build 10](https://ci.eclipse.org/openj9/view/Build/job/Build-JDK10-linux_x86-64_cmprssptrs/3/)
- [Sanity 8](https://ci.eclipse.org/openj9/view/Test/job/Test-Sanity-JDK8-linux_x86-64_cmprssptrs/2/)
- [Sanity 9](https://ci.eclipse.org/openj9/view/Test/job/Test-Sanity-JDK9-linux_x86-64_cmprssptrs/2/)
- [Sanity 10](https://ci.eclipse.org/openj9/view/Test/job/Test-Sanity-JDK10-linux_x86-64_cmprssptrs/1/)
- [Extended 8](https://ci.eclipse.org/openj9/view/Test/job/Test-Extended-JDK8-linux_x86-64_cmprssptrs/2/)
- [Extended 9](https://ci.eclipse.org/openj9/view/Test/job/Test-Extended-JDK9-linux_x86-64_cmprssptrs/2/)
- [Extended 10](https://ci.eclipse.org/openj9/view/Test/job/Test-Extended-JDK10-linux_x86-64_cmprssptrs/2/)

**Comment by AdamBrousseau** [comment]
The builds mentioned above are currently configured to point at my fork, once this is merged I will update them but we won't be broken in between.

@vsebe for review

I have yet to add the PR builds on the UI

File: buildenv/jenkins/variables/defaults.yml [hunk 1]
```diff
@@ -130,3 +130,25 @@ aix_ppc-64_cmprssptrs:
     8: '--with-cups-include=/opt/freeware/include --with-extra-ldflags=-lpthread --with-extra-cflags=-lpthread --with-extra-cxxflags=-lpthread DF=/usr/sysv/bin/df --disable-ccache --with-jobs=8'
     9: '--with-cups-include=/opt/freeware/include --with-extra-ldflags=-lpthread --with-extra-cflags=-lpthread --with-extra-cxxflags=-lpthread DF=/usr/sysv/bin/df --disable-warnings-as-errors --with-jobs=8'
     10: '--with-cups-include=/opt/freeware/include --with-extra-ldflags=-lpthread --with-extra-cflags=-lpthread --with-extra-cxxflags=-lpthread DF=/usr/sysv/bin/df --disable-warnings-as-errors --with-jobs=8'
+#========================================#
+# Linux x86 64bits Compressed Pointers
+#========================================#
+linux_x86-64_cmprssptrs:
+  boot_jdk:
+    8: '/usr/lib/jvm/java-7-openjdk-amd64'
+    9: '/usr/lib/jvm/java-8-openjdk-amd64'
```
Comment (jdekonin) [inline buildenv/jenkins/variables/defaults.yml]: Should be /usr/lib/jvm/adoptojdk-java-80

**Comment by AdamBrousseau** [comment]
New JDK9 build with Adopt BOOT_JDK
https://ci.eclipse.org/openj9/view/Build/job/Build-JDK9-linux_x86-64_cmprssptrs/3/

**Pull request merged**